src/standard_libraries/test_test_json.py

```diff
@@ -0,0 +1,33 @@
+"""Serialization.
+
+@see: https://www.learnpython.org/en/Serialization
+
+Python provides built-in JSON libraries to encode and decode JSON.
+"""
+
+import json
+
+
+def test_json():
+    """JSON serialization."""
+
+    person_dictionary = {'name': 'Fareed', 'Company': 'Arbisoft', 'age': 22}
```

Comment:
can we add another key like `"stated_test_at": datetime.datetime(2021, 8, 6, 9, 30, 0)`?

Reply:
datetime.datetime(...) at value will give a json decode error as datetime objects cannot be at value. json can handle any of the followinf datatypes only: arrays, numbers, boolean, string, null and objects (written in curly braces with key and value pairs). if we want datetime.datetime, we can do it by str(datetime.datetime())